frameworks/elastic/tests/test_kibana.py

```diff
@@ -388,9 +389,11 @@ def test_admin_router_with_folder_name() -> None:
             insert_strict_options=False,
         )
 
-        config.check_kibana_adminrouter_integration("service/{}/".format(kibana_service_name))
         config.check_kibana_adminrouter_integration(
-            "service/{}/app/kibana".format(kibana_service_name)
+            urllib.parse.quote("service/{}/".format(kibana_service_name))
```

Comment:
Only the service names should be URL-encoded (not the whole URL path) like in [my suggestions](https://github.com/mesosphere/dcos-elastic-service/pull/38#discussion_r317525064).

Reply:
thanks, fixed